packages/flutter/lib/src/rendering/proxy_sliver.dart

```diff
@@ -369,3 +370,124 @@ class RenderSliverOffstage extends RenderProxySliver {
     ];
   }
 }
+
+/// Makes its sliver child partially transparent, driven from an [Animation].
+///
+/// This is a variant of [RenderSliverOpacity] that uses an [Animation<double>]
+/// rather than a [double] to control the opacity.
+class RenderSliverAnimatedOpacity extends RenderProxySliver {
```

Comment:
I wonder if we could share some of the code that's identical in this class and in RenderAnimatedOpacity. Maybe with a Mixin?